pkgs/development/python-modules/netbox-documents/default.nix

```
@@ -0,0 +1,45 @@
+{ lib
+, buildPythonPackage
+, fetchFromGitHub
+, setuptools
+, pytestCheckHook
+, drf-extra-fields
+, netbox
+}:
+
+buildPythonPackage rec {
+  pname = "netbox-documents";
+  version = "0.6.3";
+  pyproject = true;
+
+  src = fetchFromGitHub {
+    owner = "jasonyates";
+    repo = "netbox-documents";
+    rev = "v${version}";
+    hash = "sha256-BQ33eJAp0Hnc77Mvaq9xAcDqz15fkdCwQ7Q46eOOmaI";
+  };
+
+  build-system = [
+    setuptools
+  ];
+
+  nativeCheckInputs = [
+    pytestCheckHook
+  ];
+
+  propagatedBuildInputs = [
+    drf-extra-fields
+  ];
+
+  checkInputs = [
+    netbox
+  ];
+
```

```suggestion
  # there are no tests
  doCheck = false;
  
  # also, we can't enable pythonImportsCheck probably because of the plugin system
```
Could you disable tests explicitly?

I'm not sure why `pythonImportsCheck` fails, but I guess because of the plugin system.
I have seen the similar problem with the sphinx plugin, which is the documentation tool.

---

I thought this would be a solution to it, which is why I included this also in the first version. See https://github.com/NixOS/nixpkgs/pull/301273#discussion_r1583345550 and https://github.com/NixOS/nixpkgs/pull/312866#discussion_r1609892185